common/source/docs/common-simple-object-avoidance.rst

```diff
@@ -0,0 +1,108 @@
+.. _common-simple-object-avoidance:
+
+[copywiki destination="copter,rover"]
+=======================
+Simple Object Avoidance
+=======================
+
+Copter supports simple object avoidance horizontally and upward, while Rover avoids simple objects by only stopping. Both use rangefinders such as:
+
+- 360 degree lidar including the :ref:`Lightware SF40C <copter:common-lightware-sf40c-objectavoidance>`, :ref:`TeraRanger Tower <copter:common-teraranger-tower-objectavoidance>` or :ref:`RPLidarA2/A3 <copter:common-rplidar-a2>`
+- Any of the other supported :ref:`lidars <common-rangefinder-landingpage>` .As of ArduPilot firmware versions 4.0 and higher, up to 9 rangefinders can be used for object avoidance. See :ref:`common-rangefinder-setup` for more information.
+- Sensors capable of providing `MAVLink Distance Sensor <https://mavlink.io/en/messages/common.html#DISTANCE_SENSOR>`__ messages (like `OpenKai with a 3D camera <https://www.youtube.com/watch?v=qk_hEtRASqg>`__)
+
+..  youtube:: BDBSpR1Dw_8
+    :width: 100%
+
+
+In addition, Simple Object Avoidance can use Geo-Fences and Fence Beacons as proximity sensors as detemined by the setting of the :ref:`AVOID_ENABLE<AVOID_ENABLE>` parameter.
+ 
+See :ref:`common-rangefinder-setup` for general rangefinder setup information.
+
+Simple Object Avoidance Algorithms
+==================================
+
+Horizontal object avoidance works in :ref:`AltHold <altholdmode>` and :ref:`Loiter <loiter-mode>` modes.  Upward object avoidance works in all modes except Stabilize and Acro.
+
+.. warning:: Only one proximity sensor source can be enabled, using either a 360 degree lidar or up to 9 rangefinders. While a second proximity sensor appears in the parameters list, it is not currently functional
+
+.. note:: For Object Avoidance in AUTO and GUIDED modes in Copter and Rover, see :ref:`common-oa-bendyruler` or :ref:`common-oa-dijkstras`
+
+Details on how simple object avoidance is implemented for Copter in ALTHOLD and LOITER mode can be found :ref:`here in the developer wiki <dev:code-overview-object-avoidance>` and involves assessing all the objects detected, in all reported quadrants, and adds control input to the pilot's,trying to move away from the aggregate threat.
```

Comment:
To be a little pedantic, it only moves away from objects in AltHold.  In Loiter it will stop before it hits things but will sadly never move away from them (this is on the to-do list but it's tricky).

Reply:
fixed it anyway